tests/test_generate_assessment_schema.py

```diff
@@ -12,56 +12,59 @@
 # in these tests ensures that it is always up to date with what is defined in the framework's yaml files... future work
 # to look at.
 
-# a callable so we can be sure we get a clean copy
-_definite_pass_g9_declaration = lambda: {
-    # discretionary
-    "GAAR": False,
-    "bankrupt": False,
-    "confidentialInformation": False,
-    "conflictOfInterest": False,
-    "distortedCompetition": False,
-    "distortingCompetition": False,
-    "environmentalSocialLabourLaw": False,
-    "graveProfessionalMisconduct": False,
-    "influencedContractingAuthority": False,
-    "misleadingInformation": False,
-    "seriousMisrepresentation": False,
-    "significantOrPersistentDeficiencies": False,
-    "taxEvasion": False,
-    "unspentTaxConvictions": False,
-    "witheldSupportingDocuments": False,
 
-    # mandatory
-    "10WorkingDays": True,
-    "MI": True,
-    "accurateInformation": True,
-    "accuratelyDescribed": True,
-    "canProvideFromDayOne": True,
-    "conspiracy": False,
-    "corruptionBribery": False,
-    "employersInsurance": u"Not applicable - your organisation does not need employer\u2019s liability insurance "
-                          u"because your organisation employs only the owner or close family members.",
-    "environmentallyFriendly": True,
-    "equalityAndDiversity": True,
-    "fraudAndTheft": False,
-    "fullAccountability": True,
-    "helpBuyersComplyTechnologyCodesOfPractice": True,
-    "informationChanges": True,
-    "offerServicesYourselves": True,
-    "organisedCrime": False,
-    "payForWhatUse": True,
-    "proofOfClaims": True,
-    "publishContracts": True,
-    "readUnderstoodGuidance": True,
-    "servicesDoNotInclude": True,
-    "servicesHaveOrSupport": True,
-    "termsAndConditions": True,
-    "termsOfParticipation": True,
-    "terrorism": False,
-    "understandHowToAskQuestions": True,
-    "understandTool": True,
-    "unfairCompetition": True,
-}
+def _definite_pass_g9_declaration():
```

Comment:
What is the rule here?  "Don't use lambdas if you can avoid it"?

Reply:
The rule is something about not saving lambdas to variables.

Reply:
https://pycodestyle.readthedocs.io/en/latest/intro.html#error-codes
`E731: do not assign a lambda expression, use a def`

https://www.python.org/dev/peps/pep-0008/#programming-recommendations

It's to do with tracebacks being clearer.